### PR TITLE
Feat: Added authentication

### PR DESCRIPTION
**Pull request by Mayor-Rhymes**
@ArinzeJeffrey-droid You would notice some inconsistencies in my imports. Some of them use the usual relative paths and some of them used alias. vscode started acting out since like Thursday probably because I had not shutdown my pc in over a week. Things are okay now though.

### CODE REVIEW

File: src/components/nextauthprovider.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+export default function NextAuthProvider({children}: {children: ReactNode}) {
+
+     return (
+        <SessionProvider>
+            {children}
+        </SessionProvider>
+     )
+}
```
Comment (ArinzeJeffrey-droid) [inline src/components/nextauthprovider.tsx]: Formatting needed here 😃

**Comment by Mayor-Rhymes** [comment]
@ArinzeJeffrey-droid Also, by the time you see this, I would have sent the ngrok link to your whatsapp. Thanks.

File: next.config.js [hunk 2]
```diff
@@ -1,4 +1,15 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    images: {
+        remotePatterns: [
+          {
+            protocol: 'https',
+            hostname: 'lh3.googleusercontent.com',
+            port: '',
+            // pathname: '/account123/**',
```
Comment (ArinzeJeffrey-droid) [inline next.config.js]: If the commented out code is not needed maybe get rid of it?

**Review by ArinzeJeffrey-droid — APPROVED** [review]
Looks good, I just left a few comments about cleaning up and formatting.
LGTM 🚀

File: src/components/nextauthprovider.tsx (hunk 1 above)
Comment (Mayor-Rhymes) [reply src/components/nextauthprovider.tsx]: Noted.

File: next.config.js (hunk 2 above)
Comment (Mayor-Rhymes) [reply next.config.js]: Noted

**Pull request merged**